docs/src/add-services/elasticsearch.md

```diff
@@ -22,11 +23,23 @@ See the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsea
 
 ## Supported versions
 
-Due to a licensing change, Elasticsearch versions after 7.10 aren't supported.
-For newer versions, use [OpenSearch](./opensearch.md) instead.
-To switch to OpenSearch, follow the same procedure as for [upgrading](#upgrading).
+Versions 7.10 and older are available in your projects for free,
+but they're at their end of life and are no longer receiving security updates from upstream.
+Due to a licensing change, newer versions of Elasticsearch aren't included in any Platform.sh plan.
 
-{{% image-versions-legacy "elasticsearch" %}}
+To avoid the additional cost of subscribing to a recent version of Elasticsearch,
+you can switch to one of the latest, free versions of [OpenSearch](./opensearch.md).
+To do so, follow the same procedure as for [upgrading](#upgrading).
+
+If you want to keep using Elasticsearch,
+you need to add a recent version at an additional cost [through Sales](https://platform.sh/contact/).
```

Comment:
When reading this sentence, it sounds strange to me.
"If you want to keep using Elasticsearch, you need to add it to your plan at an additional cost through Sales" ?